test_data/asm/simple_sum.asm

```diff
@@ -33,7 +37,7 @@ machine Main {
 
         start:
         jmpz CNT, check;
-        A <=X= A + ${ ("input", CNT + 1) };
+        A <=X= A + ${ ("input", std::conv::int(CNT) + 1) };
```

Comment:
The vm-to-constrained conversion replaces free inputs in assembly by functions of the current row index.
This means `("input", std::conv::int(CNT) + 1)` is turned into a witness column with the following query function:
```
witness X_free_input query(i) { ("input", std::conv::int(CNT(i)) + 1) }
```
Since `CNT` has type `int -> fe` (it is a column), we do the conversion here. I think `CNT + std::conv::fe(1)` could also work in this case, though.

Reply:
ok